pynecone/utils.py

```diff
@@ -622,6 +623,10 @@ def run_frontend(app: App, root: Path, port: str):
     # Set up the frontend.
     setup_frontend(root)
 
+    # start watching asset folder
+    asset_watch = AssetFolderWatch(root)
+    asset_watch.start()
```

Comment:
Do we need to stop the watch? For example when they ctrl+c, will this still be watching?

Reply:
We don't need to do that. 
This is a daemon thread so will be killed when the main thread is exited even if it is caused with ctrl+c.